### DISCUSSION TITLE
One monitor

### DISCUSSION THREAD

**Discussion by HamzaHajeir** [debauchee/barrier#1909] [Q&A]
Hi there,

Do I really need two real monitors to work with barrier?

I've a setup where one monitor is connected to the server, and the client is running too, but I can't find the client at the server-side.
Both are connected to the same LAN.

Client:
![image](https://user-images.githubusercontent.com/32960423/222529892-157f40fa-d39d-43dc-9a46-382c52e2aa1f.png)

Server:
![image](https://user-images.githubusercontent.com/32960423/222529930-539f8960-4793-464f-bb52-33e3b7c8a496.png)

**Comment by DeepYogurt** [comment]
The issue is that Barrier needs a monitor on each system and you share a keyboard and mouse between all systems, not the video. Am pretty sure if you added a second monitor things would work as you were expecting. Good luck...
